src/config.rs

```diff
@@ -73,6 +75,16 @@ pub(crate) struct CameraConfig {
     #[validate(range(min = 0, max = 31, message = "Invalid channel", code = "channel_id"))]
     #[serde(default = "default_channel_id")]
     pub(crate) channel_id: u8,
+
+    #[validate(range(min = 0, max = 31, message = "Invalid device number", code = "v4ldevice"))]
+    pub(crate) v4ldevice: u8,
```

Comment:
This means we must all supply a number for all camera I would recommend it be a `Option<u8>` with a default value of `None` so that it won't create a v4l if its not in the config.